routes/api/v1/favorite_locations.js

```diff
@@ -29,9 +29,10 @@ router.get("/", async (req, res, next) => {
     if(!user) {
       res.status(401).send(JSON.stringify('Unauthorized'))
     } else {
-      const favorites = await Favorite.find({where: {UserId: user.id}})
+      eval(pry.it)
+      const favorites = await Favorite.findAll({where: {UserId: user.id}})
```

Comment:
When pry stops executing, and you enter line 33 in the pry environment, it no longer knows that you are executing in an asynchronous function. If you were to move your `eval(pry.it)` after line 33, pry would know the value of favorites.

Reply:
That worked perfectly. Thank you!